mlrun/__init__.py

```diff
@@ -61,6 +61,18 @@ def get_version():
     mlconf.remote_host = mlconf.remote_host or igz_domain
 
 
+_data_stores = None
+
+
+def get_data_stores(secrets=None, override=False):
+    """singleton mlrun data store client"""
```

Comment:
`store_manager` is already cached and is a singleton, why this function is needed ? 
also setting the secret is not an expensive operation (like HTTPRunDB's connect), I don't see a reason not doing it (what if secrets were changed ?)

Reply:
fixed